1.0/docs/start/first-element/step-2.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: default
+type: start
+shortname: Start
+title: "Step 2: Add Local DOM"
+subtitle: Build your first Polymer element
+---
+
+<style>
+.caption + pre,.caption + .prettyprint {
+  margin-top: 0px;
+}
+</style>
+
+Next, you'll create a simple element that displays an icon.
+
+In this step, you'll learn about:
+
+*   Creating a custom element using Polymer.
+*   Working with local DOM.
+
+
+<div class="alert alert-info"><strong>Learn more: local DOM.</strong> Local DOM
+lets you add a <em>scoped</em> DOM tree inside an element, with local styles and
+markup that are decoupled from the rest of the web page. The element containing
+the local DOM tree is called a <em>host element</em>. To learn more, see <a
+href="https://www.polymer-project.org/1.0/docs/devguide/local-dom.html">Local
+DOM</a> in the Polymer library docs.
+</div>
+
+## Edit icon-toggle.html
+
+Click on `icon-toggle.html `in the root of the Chrome Dev Editor
+navigation panel. This file contains the skeleton of a custom element.
+
+Unlike most HTML files, this file <em>won't display anything if you load it in a
+browser</em>—it just <em>defines</em> a new element. The demo <em>imports</em>
+`icon-toggle.html` so it can use the `&lt;icon-toggle&gt;`
+element. As you add features to the element in the following steps, they'll show
+up in the demo.
+
+Start by taking a look at the existing code:
+
+
+#### Starting code—HTML imports
+{: .caption }
+
+<pre class="prettyprint">
+&lt;link rel="import"
+  href="../polymer/polymer.html">
+&lt;link rel="import"
+  href="../iron-icon/iron-icon.html">
+</pre>
+
+Key information:
+
+*   The `link rel="import"` element is an <em>HTML import</em>, a new
+    way of including resources in an HTML file.
+*   These lines import the Polymer library and another custom element called
+    `iron-icon` that you'll use later in this step.
+
+Next is the definition of the element itself:
+
+#### Starting code—local DOM template
+{: .caption }
+
+
+<pre class="prettyprint">
+&lt;dom-module id="icon-toggle">
+  &lt;template>
+    &lt;style>
+      /* local styles go here */
+      :host {
+        display: inline-block;
+      }
+    &lt;/style>
+    &lt;!-- local DOM goes here -->
+    &lt;span>Not much here yet.&lt;/span>
+  &lt;/template>
+</pre>
+
+Key information:
+
+*   The `<dom-module>` tag is an optional part of the element
```

Comment:
I know technically it's optional but I figure for 99% of use cases folks will want to use it. Mentioning that it's optional raises the question when do I use it and when do I not?